src/EdozoLogo2/EdozoLogo2.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { ProductName, StyledLogo } from './EdozoLogo2.styles';
+
+export interface Props {
+  appName?: string;
+  colour?: 'Maps' | 'Occupiers' | 'Insight';
```

Comment:
Also, in code I think we should stick to the American spelling (same as when it's required in the code) otherwise it can be confusing